samples/sensor/tmp100/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.13.1)
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(tmp100)
+
+FILE(GLOB app_sources src/*.c)
+target_sources(app PRIVATE ${app_sources})
+##############################
+# Below line is new addition #
+##############################
+add_subdirectory(../../../drivers/sensor/tmp100 tmp100)
```

Comment:
This is not how drivers should be added.

The drivers/sensor/tmp100/CMakeLists.txt should be "add_subdirectory'd" from drivers/sensor/CMakeLists.txt.